core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -2470,6 +2485,7 @@ private CompiledQuery sqlShow(SqlExecutionContext executionContext) throws SqlEx
                     return compiledQuery.of(factory);
                 }
                 Misc.free(factory);
+                // TODO: pass to parseUnexpected() instead
```

Comment:
nit: can we do something about this TODO rather than leaving it here? Seems like a trivial one.

Reply:
i removed todo here: https://github.com/questdb/questdb/pull/3145/commits/d3c8ad91ac910eecff0fbd8fc51726100b3c51fb

`parseUnexpected` is on SqlParser and cannot be made static